diff --git a/include/configs/wandboard.h b/include/configs/wandboard.h
index d2e7e45..41ad582 100644
--- a/include/configs/wandboard.h
+++ b/include/configs/wandboard.h
@@ -107,7 +107,7 @@
 	"ip_dyn=yes\0" \
 	"mmcdev=" __stringify(CONFIG_SYS_MMC_ENV_DEV) "\0" \
 	"mmcpart=1\0" \
-	"mmcroot=/dev/mmcblk0p2 rootwait rw\0" \
+	"mmcroot=/dev/mmcblk2p2 rootwait rw\0" \
 	"update_sd_firmware_filename=u-boot.imx\0" \
 	"update_sd_firmware=" \
 		"if test ${ip_dyn} = yes; then " \
